.core_files.yaml

```
@@ -102,12 +102,17 @@ tests: &tests
   - codecov.yaml
   - requirements_test_pre_commit.txt
   - requirements_test.txt
+  - tests/auth/**
+  - tests/backports/*
   - tests/common.py
   - tests/conftest.py
+  - tests/helpers/*
   - tests/ignore_uncaught_exceptions.py
   - tests/mock/*
+  - tests/scripts/**
```

There are no sub directories in scripts, but I don't know if this is important.
```suggestion
  - tests/scripts/*
```